src/NetworkConfiguration.py

```diff
@@ -8,7 +8,7 @@
     'MAINNET': {'NAME': 'MAINNET', 'NB_FREEZE_CYCLE': 5, 'BLOCK_TIME_IN_SEC': 60, 'BLOCKS_PER_CYCLE': 4096,
                 'BLOCKS_PER_ROLL_SNAPSHOT': 256},
     'ALPHANET': {'NAME': 'ALPHANET', 'NB_FREEZE_CYCLE': 3, 'BLOCK_TIME_IN_SEC': 30, 'BLOCKS_PER_CYCLE': 2048,
```

Comment:
Maybe we should rename those accrding to how tezos client/protocol devs are handling the naming? We release with every new protocol version and make it a "pre release todo" to rename those according to the new names?

Reply:
I'm ok with that. It does make things confusing to say `-N ALPHANET` when you want to test against delphinet. Do we need to keep zeronet? Does anyone use that?

Reply:
I think zeronet is not running right now (http://tezos.gitlab.io/introduction/test_networks.html). Let's keep these always updated only with the new and running test networks!